app/bundles/FormBundle/Model/FormModel.php

```diff
@@ -785,8 +785,9 @@ public function getFormScript(Form $form)
             $theme .= '|';
         }
 
+        //TODO: add a check to see if the theme exists before rendering
```

Comment:
@mabumusa1 and I discussed via PM that he will take care of this todo when removing the TemplatingHelper